pkgs/by-name/co/corectrl/package.nix

```diff
@@ -18,32 +19,33 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "corectrl";
-  version = "1.4.3";
+  version = "1.5.1";
 
   src = fetchFromGitLab {
     owner = "corectrl";
     repo = "corectrl";
-    rev = "refs/tags/v${finalAttrs.version}";
-    hash = "sha256-rQibIjLmSnkA8jk6GOo68JIeb4wZq0wxXpLs3zsB7GI=";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-NwGrvDqImiyPc3AsL7rMwNG9na+AzZS6NvXQOc6VWHg=";
   };
 
   nativeBuildInputs = [
-    extra-cmake-modules
-    libsForQt5.wrapQtAppsHook
+    cmake
+    pkg-config
+    polkit
+    kdePackages.extra-cmake-modules
```

Comment:
Are any binaries from these two run at build time? If not, they belong to `buildInputs`

Reply:
I see. ECM has a setup hook, which is why it needs to be a native build input